nachos/src/main/java/com/hootsuite/nachos/NachoTextView.java

```diff
@@ -796,6 +805,9 @@ public void onItemClick(AdapterView<?> adapterView, View view, int position, lon
 
         editable.replace(start, end, mChipTokenizer.terminateToken(text, data));
 
+        if (onChipAddListener != null){
+            onChipAddListener.onChipAdded(getAllChips().get(getAllChips().size()-1));
```

Comment:
remember to format your code with Android Studio's code formatting. The `-1` would be spread out to make it look nicer